client/src/components/Public/Sidebar/Sidebar.scss

```diff
@@ -15,8 +15,11 @@
   }
 }
 
-.drawer.drawer-left{
-    z-index: 1030;
+.drawer{
+  z-index: 900;
+}
+.drawer.drawer-left.drawer-open{
+  z-index: 1030;
```

Comment:
Oh, my favourite random integers in `z-index` :laughing: But I understand you don't want to do binary search on that...